pkg/controller/controller_test.go

```diff
@@ -643,3 +645,53 @@ func TestDeleteVolumeGroup(t *testing.T) {
 		t.Errorf("Failed to delete volume group: %v\n", err)
 	}
 }
+func NewFakeMetricController() metrics.Controller {
+	return &fakeMetricsController{}
+}
+type fakeMetricsController struct {
+}
+
+func (fakeMetricsController) CollectMetrics(opt *pb.CollectMetricsOpts) ([]*model.MetricSpec, error) {
+	return nil,nil
+}
+
+func (fakeMetricsController) SetDock(dockInfo *model.DockSpec) {
+	return
+}
+
+func (fakeMetricsController) GetUrls() (*map[string]model.UrlDesc, error) {
+	return nil,nil
+}
+
+func TestGetMetrics(t *testing.T) {
+
+	var req = []*pb.GetMetricsOpts{
+		{InstanceId: "", StartTime: "", EndTime: ""},
+		{InstanceId: "1234", StartTime: "1234", EndTime: "1234"},
+		{InstanceId: "1234", StartTime: "1234", EndTime: "3456"},
+	}
+
+	var ctrl = &Controller{
+		metricsController: NewFakeMetricController(),
+	}
+	var response *pb.GenericResponse
+	var err error
+	for	_,r:= range req{
+	if response, err = ctrl.GetMetrics(context.Background(), r); err != nil {
+		t.Errorf("Failed to create volume, err is %v\n", err)
```

Comment:
All error msg should start with lower case letter.

Reply:
fixed, thanks